Sources/ApiVideoPlayer/Views/SelectableListView.swift

```diff
@@ -0,0 +1,129 @@
+        tableview.dataSource = self
+        tableview.register(UITableViewCell.self, forCellReuseIdentifier: cellReuseIdentifier)
+
+        addSubview(tableview)
+        bringSubviewToFront(tableview)
+    }
+
+    @available(*, unavailable)
+    required init?(coder _: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func tableView(_: UITableView, numberOfRowsInSection _: Int) -> Int {
+        listAny.count
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell: UITableViewCell = tableView.dequeueReusableCell(
+            withIdentifier: cellReuseIdentifier,
+            for: indexPath
+        )
+
+        let rowElement = listAny[indexPath.row]
+
+        if #available(iOS 14.0, *) {
+            var content = cell.defaultContentConfiguration()
+            // do if T is Float
+            if let rowElementIsFloat = rowElement as? Float {
+                content.text = rowElementIsFloat.description
+            }
+            // do if T is Locale
+            if let rowElementIsFloat = rowElement as? SubtitleLanguage {
+                content.text = rowElementIsFloat.language
+            }
+            cell.contentConfiguration = content
+        } else {
+            if let rowElementIsFloat = rowElement as? Float {
+                cell.textLabel?.text = rowElementIsFloat.description
+            }
+            // do if T is SubtitleLanguage
+            if let rowElementIsFloat = rowElement as? SubtitleLanguage {
+                cell.textLabel?.text = rowElementIsFloat.language
+            }
+        }
+
+        if let notOptionaleSelectedAny = selectedAny {
+            if type(of: rowElement) == type(of: notOptionaleSelectedAny) {
+                if rowElement == notOptionaleSelectedAny {
+                    cell.accessoryType = .checkmark
+                }
+            }
+        }
+
+        return cell
+    }
+
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        tableView.deselectRow(at: indexPath, animated: false)
+
+        let selectedRowElement = listAny[indexPath.row]
+        if selectedRowElement != selectedAny {
+            // Uncheck previous selected element
+            if let selected = selectedAny {
+                if let previousSelectedElementIndex = getRowForElement(selected) {
+                    if let previousCell = tableView
+                        .cellForRow(at: IndexPath(row: previousSelectedElementIndex, section: indexPath.section))
+                    {
+                        previousCell.accessoryType = .none
+                    }
+                }
+            }
+            selectedAny = selectedRowElement
+            if let cell = tableView.cellForRow(at: indexPath) {
+                cell.accessoryType = .checkmark
+            }
+        }
+
+        if let selectedRowIsSubtitle = selectedRowElement as? SubtitleLanguage {
```

Comment:
`SelectableListView` should not know the type of the elements. This is the principle of generic type.